AAP-16032: Docathon updates to Disconnected Installation chapter (#405)
ansible/aap-docs: downstream/modules/platform/proc-installing-the-aap-setup-bundle.adoc

// this info is duplicated here: https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/1.2/html/installing_and_upgrading_private_automation_hub/installing_online_or_offline#doc-wrapper

[id="installing-the-aap-setup-bundle_{context}"]

= Downloading and installing the {PlatformNameShort} setup bundle

[role="_abstract"]

> The “bundle” version is strongly recommended for disconnected installations because it includes the RPM content for {PlatformNameShort} as well as the default {ExecEnvShort} images that will be uploaded to your {PrivateHubName} during the installation process.

Reviewer: See the Red Hat Supplementary style guide in the [Word usage section](https://redhat-documentation.github.io/supplementary-style-guide/#_word_usage), specifically the word "recommend". Currently, it says "Avoid 'recommend'." Instead direct users to take the recommended action. 
```suggestion
Use the "bundle" version for disconnected installations because it includes  the RPM content for {PlatformNameShort} as well as the default {ExecEnvShort} images that will be uploaded to your {PrivateHubName} during the installation process.
```

.Procedure

. Download the {PlatformNameShort} setup bundle package by navigating to the link:{PlatformDownloadUrl}[{PlatformName} download] page and clicking btn:[Download Now] for the {PlatformNameShort} {PlatformVers} setup bundle.

. From {ControllerName}, untar the bundle:
+
----
$ tar xvf \
   ansible-automation-platform-setup-bundle-2.4-1.tar.gz
$ cd ansible-automation-platform-setup-bundle-2.4-1
----
+
. Edit the inventory file to include the required options:

> .. automationcontroller group

Reviewer: Maybe change these options to a bullet list instead of sequential numbering.

automationhub_pg_password='dbpassword123'
automationhub_pg_sslmode='prefer'
----

> NOTE: The inventory should be kept intact after installation because it is used for backup, restore, and upgrade functions. Consider keeping a backup copy in a secure location, given that the inventory file contains passwords.

Reviewer: Just a reminder to correct the format for this admonition in asciidoc. 

```suggestion
[NOTE]
====
The inventory must remain intact after installation because it is used for backup, restore, and upgrade functions. Consider keeping a backup copy in a secure location, given that the inventory file contains passwords.
====
```

Tweaked the wording to use stronger word than "should" (refer to [IBM Word usage for "should"](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s)

+
. Run the {PlatformNameShort} setup bundle executable as the root user:
+
----
$ sudo -i
# cd /path/to/ansible-automation-platform-setup-bundle-2.4-1
# ./setup.sh
----
+

> . When installation is complete, navigate to the Fully Qualified Domain Name (FQDN) for the {ControllerName} node that was specified in the installation inventory file.

Reviewer: Check with @dcdacosta about the FQDN. For some reason, I thought this had changed.

Reviewer 2: I'm not aware of a change with regard to FQDN? 🤔

Reviewer: Okay. I think I was peer reviewing someone else's PR and they replaced this value. Ignore my comment above.
